app/src/ui/clone-repository/clone-repository.tsx

```diff
@@ -316,11 +325,14 @@ export class CloneRepository extends React.Component<
 
   /**
    * Lookup the account associated with the clone (if applicable) and resolve
-   * the repository alias to the clone URL.
+   * the repository data.
    */
-  private async resolveCloneURL(): Promise<string | null> {
+  private async resolveCloneData(): Promise<ICloneData | null> {
     const identifier = this.state.lastParsedIdentifier
-    let url = this.state.url
+    const cloneData: ICloneData = {
```

Comment:
Rather than assigning this early and mutating this later (losing the benefit of `const`), could we remove this and `return` when we have the information we need?